checkov/kubernetes/runner.py

```diff
@@ -191,6 +199,41 @@ def run(self, root_folder, external_checks_dir=None, files=None, runner_filter=R
 
         return report
 
+    def get_graph_checks_report(self, root_folder: str, runner_filter: RunnerFilter) -> Report:
+        report = Report(self.check_type)
+        checks_results = self.run_graph_checks_results(runner_filter)
+
+        for check, check_results in checks_results.items():
+            for check_result in check_results:
+                entity = check_result["entity"]
+                entity_file_abs_path = entity.get(CustomAttributes.FILE_PATH)
+                entity_file_path = f"/{os.path.relpath(entity_file_abs_path, root_folder)}"
+                start_line = entity.get('__startline__')
+                end_line = entity.get('__endline__')
```

Comment:
is there a case when either one them is not set? Because it would break the lower part of the code

Reply:
there shouldn't be a case that one of them not set, you can see another use of them in the `run` method

Reply:
ok, then just access them directly, `entity['__startline__']` so if they shouldn't exist for some reason, then it will fail here and not a few lines lower 🙂